File: Week1/17Jan/C/prog2.c
int printf(const char*,...);
int scanf(const char*,...);

void main(){
	int unit;
	float bill;
	printf("\nEnter units consumed = ");
	scanf("%d",&unit);
	if(unit<=50)
		bill = unit * 0.5;
	else if(unit <= 150)
		bill = 25 + ((unit-50)*0.75);
	else if(unit <= 250)
		bill = 100 + ((unit-150)*1.2);
	else if(unit > 250)
		bill = 220 + ((unit-250)*1.5);
	printf("Bill = %.2f\n",bill);
}

File: Week1/17Jan/Python/prog2.py
unit = int(input("Enter units consumed = "))

if unit <= 50:
    bill = unit * 0.5
elif unit <= 150:
    bill = 25 + ((unit-50)*0.75)
elif unit <= 250:
    bill = 100 + ((unit-150)*1.2)
elif unit > 250:
    bill = 220 + ((unit-250)*1.5)
print("Bill = ",bill)
